### ISSUE TITLE
Position kf_indicator in the centre of image view

### ISSUE THREAD

**Issue by spidergears** [onevcat/Kingfisher#113]
Is there a way position the kf_indicator in the centre of imageView. 
I use this, but it annoys me that the the indicator always show outside the imageView.
I tried positioning with frame x, y and also with centre... nothing works.

**Comment by onevcat** [comment]
The indicator should be in the center of image view (https://github.com/onevcat/Kingfisher/blob/master/Kingfisher/UIImageView%2BKingfisher.swift#L284), however, since it is a setting for frame directly, it might be some cases the position gets wrong when using auto layout and do some rotation. 

May I know some context of your code? So I can get some detail on the problem to get a fix for it.

**Comment by jfizz** [comment]
I had the same issue after upgrading to the latest version. changing https://github.com/onevcat/Kingfisher/blob/master/Kingfisher/UIImageView%2BKingfisher.swift#L284 to the code below resolved the issue. i believe this post explains what is going on: http://stackoverflow.com/questions/8588248/uiview-center-uiimageview-center-but-image-view-doesnt-seem-to-be-in-centre-i

```
indicator.center = CGPointMake(CGRectGetMidX(bounds), CGRectGetMidY(bounds))
```

**Comment by onevcat** [comment]
@jfizz 
Wow, that's cool. Didn't notice that before. Would you like to send a pull request on it?

**Comment by spidergears** [comment]
@jfizz
Can you please create pull request for this. It will be really helpful.

**Comment by onevcat** [comment]
:) Let me do it.

**Comment by onevcat** [comment]
Fixed in #115
